shippable/install.sh

```diff
@@ -145,3 +145,9 @@ pip3 install -q statistics numpy
 
 pip3 install -r https://raw.githubusercontent.com/JuulLabs-OSS/mcuboot/master/scripts/requirements.txt
 
+# imgtool, needed for MCUboot to sign images (i.e. for TF-M S/NS builds)
+pip3 install -q imgtool
```

Comment:
Not related to this PR, but do we know why imgtool isn't in https://raw.githubusercontent.com/JuulLabs-OSS/mcuboot/master/scripts/requirements.txt ?

Reply:
Good point, I was actually looking into the mcu boot requirements, and plan to check with David.

Reply:
@galak it is now: https://github.com/mcu-tools/mcuboot/blob/master/scripts/requirements.txt

Reply:
though the contents are the same :)

Reply:
do you mind adding a commit to move to ` https://github.com/mcu-tools`

Reply:
yeap, sure, done